DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditAutoRefreshOptions.cshtml

```diff
@@ -32,23 +32,23 @@
                       hint-text="Which course should the learner be enrolled on when auto-refreshing?"
                       deselectable="false"
                       css-class="nhsuk-u-width-one-half"
-                      default-option="Select a field name"
+                      default-option="Select a course"
                       select-list-options="@Model.CourseOptions" />
 
       <vc:numeric-input asp-for="@nameof(Model.AutoRefreshMonths)"
                         label="Enrol on refresher (months before expiry)"
                         populate-with-current-value="true"
                         type="text"
-                        hint-text="How many months prior to expiry of the previous course should be learner be enrolled on the refresher cause."
+                        hint-text="How many months prior to expiry of the previous course should the learner be enrolled on the refresher course?"
                         css-class="nhsuk-input--width-10" />
 
       <vc:single-checkbox asp-for="@nameof(Model.ApplyLpDefaultsToSelfEnrol)"
                           label="Apply to self-enrolments"
-                          hint-text="Apply auto-refresh settings when a user has self enrolled on a course (rather than being enrolled by an administrator)" />
+                          hint-text="Apply auto-refresh settings when a user has self-enrolled on a course (rather than being enrolled by an administrator)" />
 
       <button class="nhsuk-button" type="submit">Save</button>
     </form>
 
-    <vc:cancel-link asp-controller="ManageCourse" asp-action="Index" asp-all-route-data="@cancelLinkData" />
+    <vc:back-link asp-controller="ManageCourse" asp-action="EditLearningPathwayDefaults" asp-all-route-data="@backLinkData" link-text="Go back" />

```

Comment:
The cancel link is better actually - it's consistent with the other edit pages on Course Setup.

Reply:
changed it back :)

Reply:
Sorry... I was wrong, back-link is what we want.

Reply:
removed the last commit so it's a back link again